app/assets/javascripts/admin.js.coffee

```diff
@@ -1,33 +0,0 @@
-#= require_self
```

Comment:
it's really strange, i cannot find this file. i can only find admin.coffee (don't have admin.js.coffee)